test/evaluator/Test_Evaluator_Prelude.re

```diff
@@ -3,10 +3,23 @@ open Language;
 
 module UG = Grammar.UnitGrammar;
 
-let testable_exp = (~ignore_constructor_types=?, ()) =>
+let testable_exp = (~ignore_constructor_types=false, ()) =>
   testable(
     Fmt.using(Exp.show, Fmt.string),
-    DHExp.fast_equal(~ignore_constructor_types?),
+    Equality.equality(
+      ~type_alpha=false,
+      ~exp_alpha=false,
+      ~ignore_wrappers=false,
+      ~ignore_casts=false,
+      ~ignore_constructor_types,
+      ~ignore_function_names=false,
+      ~ignore_filters=false,
+      ~ignore_unknown_provenance=false,
```

Comment:
Same issue with unknown provenance and surface representation. We're using `?` everywhere to mean unknown but not specifying a provenance.